### DISCUSSION TITLE
[Question][Hw5] What is "CallGraph.is_recursive" function intended to do?

### DISCUSSION THREAD

**Discussion by tncks0121** [prosyslab-classroom/cs524-program-analysis#307] [Q&A] [answered]
Name: Suchan Park

From the README, it seems I have to fill out `is_recursive` function in `CallGraph`. However, I am not sure what this function is expected to do. 

If I have another idea, am I allowed not to fill out this function?

**Comment by Re-st** [comment, marked as answer]
- `let is_recursive f callgraph` checks if f (a function) is recursive.
- Recursion is when a function calls itself.
- `is_recursive` is expected to use `callgraph`
- `is_recursive` is not yet used in any parts of code, but you may use it somewhere.

The answer will be updated if I can give more hints.
